### PR TITLE
Rewrite documentation - Main Concepts - Realtime Engine 

### PR DESCRIPTION
**Pull request by Aschen**
https://deploy-preview-1847--doc-kuzzle.netlify.app/guides/main-concepts/5-realtime-engine

### CODE REVIEW

**Review by Njuelle — APPROVED** [review]

File: doc/2/guides/introduction/1-what-is-kuzzle/index.md [hunk 1]
```diff
@@ -98,4 +98,4 @@ The community is also able to develop and distribute its own plugins to enrich t
 
 We federate a community of developers using Kuzzle around the world.
 
-Whether you want to ask a question about [StackOverflow](https://stackoverflow.com/questions/ask?tags=kuzzle), check out the [Kuzzle awesome list](https://github.com/kuzzleio/awesome-kuzzle) or discuss Kuzzle usage on [Discord](http://join.discord.kuzzle.io), the community and the core team will be there to help you.
+Whether you want to ask a question about [StackOverflow](https://stackoverflow.com/questions/ask?tags=kuzzle), check out the [Kuzzle awesome list](https://github.com/kuzzleio/awesome-kuzzle), watch our video on [YouTube](https://www.youtube.com/channel/UCHcEzVQoH10YSyxc7jD3SMw/videos) discuss Kuzzle usage on [Discord](http://join.discord.kuzzle.io), the community and the core team will be there to help you.
```
Comment (scottinet) [inline doc/2/guides/introduction/1-what-is-kuzzle/index.md:101]:
```suggestion
Whether you want to ask a question on [StackOverflow](https://stackoverflow.com/questions/ask?tags=kuzzle), check out the [Kuzzle awesome list](https://github.com/kuzzleio/awesome-kuzzle), watch our video on [YouTube](https://www.youtube.com/channel/UCHcEzVQoH10YSyxc7jD3SMw/videos), or discuss Kuzzle on [Discord](http://join.discord.kuzzle.io), the community and the core team will be there to help you.
```

File: doc/2/guides/main-concepts/5-realtime-engine/index.md [hunk 2]
```diff
@@ -8,99 +8,83 @@ order: 500
 
 # Realtime Engine
 
-<!-- need rewrite -->
+Kuzzle includes his **own high-performance Realtime Engine** for sending notifications to clients connected through the API.
 
-Kuzzle includes his own realtime engine for sending notifications to clients connected through the API.
-
-Realtime capabilities requires the use of a persistent communication protocol such as WebSocket or MQTT.
+Realtime capabilities requires the use of a **persistent communication protocol** such as WebSocket or MQTT.
```
Comment (scottinet) [inline doc/2/guides/main-concepts/5-realtime-engine/index.md:13]:
```suggestion
Realtime capabilities require the use of a **persistent communication protocol** such as WebSocket or MQTT.
```

File: doc/2/guides/main-concepts/5-realtime-engine/index.md [hunk 3]
```diff
@@ -8,99 +8,83 @@ order: 500
...
 
-Kuzzle's realtime engine allows you to do Pub/Sub in dedicated communication channels called rooms.
+Kuzzle's Realtime Engine allows you to do **Pub/Sub in dedicated communication channels called rooms**.
 
 The process is as follows:
- - a customer subscribes to a particular room,
- - a second customer posts a message in this room,
- - the customer subscribing to the room receives a notification.
+ - A first client subscribes to a particular room,
+ - A second client client posts a message in this room,
```
Comment (scottinet) [inline doc/2/guides/main-concepts/5-realtime-engine/index.md:25]:
```suggestion
 - A second client posts a message in that room,
```

File: doc/2/guides/main-concepts/5-realtime-engine/index.md [hunk 4]
```diff
@@ -136,72 +120,72 @@ More information about the [Document Notification format](/core/2/api/essentials
...
+ - Name of the index,
+ - Name of the collection you want to watch,
+ - Subscription filters contained in the `body` of the request.
 
 ::: info
-The specified index and collection must exist in the database to receive database notifications.
+The specified index and collection **must exist in the database** to receive database notifications.
 :::
 
 When changes occur on this collection (eg: document creation, modification or deletion), Kuzzle will send notifications to the corresponding subscribers.
```
Comment (scottinet) [inline doc/2/guides/main-concepts/5-realtime-engine/index.md:140]:
```suggestion
When changes occur on this collection (e.g. document creation, modification or deletion), Kuzzle will send notifications to the corresponding subscribers.
```

File: doc/2/guides/main-concepts/5-realtime-engine/index.md [hunk 5]
```diff
@@ -136,72 +120,72 @@ More information about the [Document Notification format](/core/2/api/essentials
...
-For example, creating a document with the [document:create](/core/2/api/controllers/document/create) method corresponds to a change in the database, so customers subscribing to notifications in this collection will be notified.
+```bash
+# Creates index and collection
+$ kourou collection:create nyc-open-data yellow-taxi
 
-```json
-{
-  "controller": "document",
-  "action": "create",
+# Subscribe to database notifications
```
Comment (scottinet) [inline doc/2/guides/main-concepts/5-realtime-engine/index.md:148]:
```suggestion
# Subscribes to database notifications
```

File: doc/2/guides/main-concepts/5-realtime-engine/index.md [hunk 6]
```diff
@@ -232,17 +216,17 @@ Customers subscribing to the changes in this collection will receive the followi
 
 More information about the [Document Notification format](/core/2/api/essentials/notifications#documents-changes-messages)
 
-## Subscription filters
+## Subscription Filters
 
-When a customer subscribes to realtime notifications, whether in Pub/Sub or Database Notification, he can specify a set of subscription filters.
-These filters allow the customer to tell Kuzzle exactly which documents they are interested in and only receive notifications about them.
+When a client subscribes to realtime notifications, whether in Pub/Sub or Database Notification, he can **specify a set of subscription filters**.
```
Comment (scottinet) [inline doc/2/guides/main-concepts/5-realtime-engine/index.md:221]:
```suggestion
When a client subscribes to realtime notifications, whether in Pub/Sub or Database Notification, they can **specify a set of subscription filters**.
```

File: doc/2/guides/main-concepts/5-realtime-engine/index.md [hunk 7]
```diff
@@ -399,35 +395,26 @@ If a second customer subscribes to the same notifications, then the customer wil
...
+  "collection": "yellow-taxi",
+  "body": {}
+}'
 ```
 
-If another client publishes a message in the room specifying volatile data:
-
-```json
- {
+Then, in another terminal you can publishes a message in the room specifying volatile data:
```
Comment (scottinet) [inline doc/2/guides/main-concepts/5-realtime-engine/index.md:415]:
```suggestion
Then, in another terminal you can publish a message in the room specifying volatile data:
```

**Review by scottinet — CHANGES_REQUESTED** [review]

**Review by scottinet — APPROVED** [review]

**Review by alexandrebouthinon — APPROVED** [review]

File: doc/2/guides/main-concepts/5-realtime-engine/index.md [hunk 8]
```diff
@@ -8,99 +8,83 @@ order: 500
 
 # Realtime Engine
 
-<!-- need rewrite -->
+Kuzzle includes his **own high-performance Realtime Engine** for sending notifications to clients connected through the API.
```
Comment (xbill82) [inline doc/2/guides/main-concepts/5-realtime-engine/index.md:11]:
```suggestion
Kuzzle is shipped with its own **high-performance Realtime Engine** for sending notifications to clients connected through the API.
```

File: doc/2/guides/main-concepts/5-realtime-engine/index.md [hunk 9]
```diff
@@ -439,15 +426,10 @@ If another client publishes a message in the room specifying volatile data:
...
+ }'
 ```
 
-Payload to send with wscat in another terminal:
-```json
-{"controller":"realtime","action":"publish","index":"nyc-open-data","collection":"yellow-taxi","body":{"name":"Manwë","licence":"B","car":"berline","position":{"lat":43.6073913,"lon":3.9109057}},"volatile":{"senderName":"Eru Ilúvatar"}}
-```
-
-Each customer who subscribes to the room will receive the following notification:
+Each client who subscribes to the room will receive the following notification:
```
Comment (xbill82) [inline doc/2/guides/main-concepts/5-realtime-engine/index.md:432]:
```suggestion
Each client subscribing to the room will receive the following notification:
```

File: doc/2/guides/main-concepts/5-realtime-engine/index.md [hunk 10]
```diff
@@ -480,3 +462,7 @@ Each customer who subscribes to the room will receive the following notification
 }
 </pre>
 </details>
+
+::: info
+SDKs and Kourou use volatile data to send additional informations like the client name or version.
```
Comment (xbill82) [inline doc/2/guides/main-concepts/5-realtime-engine/index.md:467]:
```suggestion
SDKs and Kourou use volatile data to send additional information like the client name or version.
```

**Review by xbill82 — APPROVED** [review]
Wording nitpikking

**Pull request merged**